diff --git a/node_modules/@ethersproject/contracts/lib.esm/index.js b/node_modules/@ethersproject/contracts/lib.esm/index.js
index 9323575..670f9c8 100644
--- a/node_modules/@ethersproject/contracts/lib.esm/index.js
+++ b/node_modules/@ethersproject/contracts/lib.esm/index.js
@@ -321,7 +321,7 @@ export class Contract {
                     defineReadOnly(this.filters, name, this.filters[filters[0]]);
                 }
                 else {
-                    logger.warn(`Duplicate definition of ${name} (${filters.join(", ")})`);
+                    // logger.warn(`Duplicate definition of ${name} (${filters.join(", ")})`);
                 }
             });
         }
@@ -376,7 +376,7 @@ export class Contract {
         Object.keys(uniqueFunctions).forEach((name) => {
             const signatures = uniqueFunctions[name];
             if (signatures.length > 1) {
-                logger.warn(`Duplicate definition of ${name} (${signatures.join(", ")})`);
+                // logger.warn(`Duplicate definition of ${name} (${signatures.join(", ")})`);
                 return;
             }
             if (this[name] == null) {
diff --git a/node_modules/@ethersproject/contracts/lib/index.js b/node_modules/@ethersproject/contracts/lib/index.js
index 1466c01..f671c39 100644
--- a/node_modules/@ethersproject/contracts/lib/index.js
+++ b/node_modules/@ethersproject/contracts/lib/index.js
@@ -366,7 +366,7 @@ var Contract = /** @class */ (function () {
                     properties_1.defineReadOnly(_this.filters, name, _this.filters[filters[0]]);
                 }
                 else {
-                    logger.warn("Duplicate definition of " + name + " (" + filters.join(", ") + ")");
+                    // logger.warn("Duplicate definition of " + name + " (" + filters.join(", ") + ")");
                 }
             });
         }
@@ -421,7 +421,7 @@ var Contract = /** @class */ (function () {
         Object.keys(uniqueFunctions).forEach(function (name) {
             var signatures = uniqueFunctions[name];
             if (signatures.length > 1) {
-                logger.warn("Duplicate definition of " + name + " (" + signatures.join(", ") + ")");
+                // logger.warn("Duplicate definition of " + name + " (" + signatures.join(", ") + ")");
                 return;
             }
             if (_this[name] == null) {
