diff --git a/node_modules/@ethersproject/contracts/lib/index.js b/node_modules/@ethersproject/contracts/lib/index.js
index a12a4da..3db5369 100644
--- a/node_modules/@ethersproject/contracts/lib/index.js
+++ b/node_modules/@ethersproject/contracts/lib/index.js
@@ -362,7 +362,7 @@ var Contract = /** @class */ (function () {
                     properties_1.defineReadOnly(_this.filters, name, _this.filters[filters[0]]);
                 }
                 else {
-                    logger.warn("Duplicate definition of " + name + " (" + filters.join(", ") + ")");
+                    // logger.warn("Duplicate definition of " + name + " (" + filters.join(", ") + ")");
                 }
             });
         }
@@ -417,7 +417,7 @@ var Contract = /** @class */ (function () {
         Object.keys(uniqueFunctions).forEach(function (name) {
             var signatures = uniqueFunctions[name];
             if (signatures.length > 1) {
-                logger.warn("Duplicate definition of " + name + " (" + signatures.join(", ") + ")");
+                // logger.warn("Duplicate definition of " + name + " (" + signatures.join(", ") + ")");
                 return;
             }
             if (_this[name] == null) {
